modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/internal/upgrade/v1_8_1/AssetListEntryUpgradeProcess.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.asset.list.internal.upgrade.v1_8_1;
+
+import com.liferay.document.library.kernel.model.DLFileEntry;
+import com.liferay.document.library.kernel.model.DLFileEntryTypeConstants;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.model.GroupConstants;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Adolfo Pérez
+ */
+public class AssetListEntryUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
```

Comment:
Do you need to upgrade the permissions too.
`
Caused by: com.liferay.portal.kernel.exception.NoSuchResourcePermissionException: {companyId=20097, name=com.liferay.document.library.kernel.model.DLFileEntryType, primKey=44301, scope=4}
`

Reply:
@AliciaGarciaGarcia Can you post the full stacktrace (or reproduction steps)? Permissions should not be checked for the basic document type.

Reply:
Nevermind, I have everything I need in your followup comment.